openshift/templates/visual-verifier/visual-verifier-deploy.param

```diff
@@ -1,22 +1,22 @@
 #=========================================================
 # OpenShift template parameters for:
 # Component: .
-# Template File: ../openshift/templates/visual-verifier/visual-verifier-deploy.json
+# Template File: ../openshift/templates/visual-verifier/visual-verifier-deploy.yaml
 #=========================================================
 NAME=email-verification-demo
 APP_NAME=email-verification-demo
 APP_GROUP=email-verification-demo
 SUFFIX=
 ROLE=email-verification-demo
-APPLICATION_DOMAIN=verified-email-authentication.pathfinder.gov.bc.ca
-IMAGE_NAMESPACE=devex-von-image-tools
-TAG_NAME=latest
+APPLICATION_DOMAIN=bcgov-verified-email-authentication-dev.apps.silver.devops.gov.bc.ca
+IMAGE_NAMESPACE=4a9599-tools
+TAG_NAME=dev
 APP_FILE=
 APP_CONFIG=
 APP_MODULE=
 # DJANGO_SECRET_KEY=[\w]{50}
 DJANGO_DEBUG=False
-VERIFIER_NAME=Verified Email Authentication
+VERIFIER_NAME=VC Visual Verifier
```

Comment:
Label should be kept the same as old one - you mat want to add BCGov so it reads `BCGov Verified Email Authentication`

Reply:
changed verifier name and changed namespace to a99fd4 in param files